internal/impl/config.go

```diff
@@ -106,7 +105,7 @@ func readConfigFromURL(url *url.URL) (*Config, error) {
 }
 
 func upgradeConfig(cfg *Config, absFilePath string) error {
-	if notUpgrade, _ := strconv.ParseBool(os.Getenv(env.DevboxDoNotUpgradeConfig)); notUpgrade {
+	if env.NotUpgradeConfig() {
```

Comment:
nit: `env.DoNotUpgradeConfig` reads more naturally

Reply:
Okay, I'll update it in the next PR

Reply:
See https://github.com/jetpack-io/devbox/pull/954